packages/cmf/src/cmfConnect.js

```diff
@@ -34,14 +34,24 @@ import React, { createElement } from 'react';
 import hoistStatics from 'hoist-non-react-statics';
 import ImmutablePropTypes from 'react-immutable-proptypes';
 import { connect } from 'react-redux';
+import omit from 'lodash/omit';
 import api from './api';
 import deprecated from './deprecated';
+import CONSTANT from './constant';
 
 import { statePropTypes, initState, getStateAccessors, getStateProps } from './componentState';
 import { mapStateToViewProps } from './settings';
 
 let newState;
 
+function serializeEvent(event) {
+	if (event.persist) {
+		debugger;
```

Comment:
why do you add a debugger here. Could you replace by console.error/warn is really necessary ?

Reply:
my bad :(